### PR TITLE
2450-Introduce-Cell-selection-in-FastTable

### LINKED ISSUE #2419: Use PharoShortcut in FastTable instead of hardcoding select all shortcut

### LINKED ISSUE #2450: Introduce Cell selection in FastTable
I'll do a PR to introduce a cell selection feature in FastTable.

It will be implemented via a strategy design pattern and will go with tests.

### PR DESCRIPTION
**Pull request by jecisc**
Introduce a cell selection feature in FastTable.

It is implemented via a strategy design pattern and will go with tests.

Fixes #2450 
Fixes #2419

### CODE REVIEW

File: src/Morphic-Widgets-FastTable/FTMultipleSelection.class.st [hunk 1]
```diff
@@ -14,51 +14,43 @@ FTMultipleSelection >> isMultiple [
...
 
 { #category : #accessing }
-FTMultipleSelection >> selectRowIndex: rowIndex event: event [
-	event shiftPressed ifTrue: [ ^ self selectRowIndexesUpTo: rowIndex ].
-	event commandKeyPressed ifTrue: [ ^ self selectAppendingRowIndex: rowIndex ].
-	self toggleRowIndex: rowIndex
-
+FTMultipleSelection >> selectIndex: index event: event [
+	event shiftPressed ifTrue: [ ^ self selectIndexesUpTo: index ].
+	OSPlatform current isMacOS
```
Comment (StephanEggermont) [inline src/Morphic-Widgets-FastTable/FTMultipleSelection.class.st:31]: Looks like this should be a method on the current platform, OSPlatform current appendSelectionFrom: event or so? (needs a better name than that)

File: src/Morphic-Widgets-FastTable/FTSelectableMorph.class.st [hunk 2]
```diff
@@ -0,0 +1,58 @@
+"
+Description
+--------------------
+
+I am a morph that can be selected by a user in a FastTable.
+
+I manage the color changes whe the user is doing a mouse over or a selection.
```
Comment (StephanEggermont) [inline src/Morphic-Widgets-FastTable/FTSelectableMorph.class.st]: when

File: src/Morphic-Widgets-FastTable/FTMultipleSelection.class.st (hunk 1 above)
Comment (jecisc) [reply src/Morphic-Widgets-FastTable/FTMultipleSelection.class.st:31]: I already opened an issue about that.

File: src/Morphic-Widgets-FastTable/FTSelectableMorph.class.st (hunk 2 above)
Comment (jecisc) [reply src/Morphic-Widgets-FastTable/FTSelectableMorph.class.st]: Thanks

**Comment by Ducasse** [comment]
Broken tests are not related.

**Pull request merged**